libNeonDomain/include/Neon/domain/internal/mGrid/mPartition.h

```diff
@@ -79,26 +81,57 @@ class mPartition : public Neon::domain::internal::bGrid::bPartition<T, C>
     NEON_CUDA_HOST_DEVICE inline auto childVal(const Cell& childCell,
                                                int         card = 0) const -> const T&;
 
+    /**
+     * Check if the cell is refined i.e., has children 
+     * @param cell the cell i.e., parent at which the children are checked 
+     * @return 
+    */
+    NEON_CUDA_HOST_DEVICE inline auto hasChildren(const Cell& cell) const -> bool;
+
+    /**
+     * Check if a neighbor to 'cell' has children 
+     * @param cell 
+     * @param cell 
```

Comment:
The documentation of the second parameter should be updated. 
I'd suggest adding the documentation directly aside of the function/method parameters.